AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DepthOfFieldAdded.py

```diff
@@ -30,15 +31,141 @@ class Tests:
     depth_of_field_component = (
         "Entity has a DepthOfField component",
         "Entity failed to find DepthOfField component")
-    depth_of_field_disabled = (
+    depth_of_field_component_disabled = (
         "DepthOfField component disabled",
-        "DepthOfField component was not disabled.")
+        "DepthOfField component was not disabled")
+    depth_of_field_component_enabled = (
+        "DepthOfField component enabled",
+        "DepthOfField component was not enabled")
     post_fx_component = (
         "Entity has a Post FX Layer component",
         "Entity did not have a Post FX Layer component")
-    depth_of_field_enabled = (
-        "DepthOfField component enabled",
-        "DepthOfField component was not enabled.")
+    cameraentityid_override_on = (
+        "CameraEntityId Override turned on",
+        "CameraEntityId Override could not be turned on")
+    cameraentityid_override_off = (
+        "CameraEntityId Override turned off",
+        "CameraEntityId Override could not be turned off")
+    enabled_override_on = (
+        "Enabled Override turned on",
+        "Enabled Override could not be turned on")
+    enabled_override_off = (
+        "Enabled Override turned off",
+        "Enabled Override could not be turned off")
+    qualitylevel_override_on = (
+        "QualityLevel Override turned on",
+        "QualityLevel Override could not be turned on")
+    qualitylevel_override_off = (
+        "QualityLevel Override turned off",
+        "QualityLevel Override could not be turned off")
+    aperturef_override_set_to_1 = (
+        "ApertureF Override set to 1",
+        "ApertureF Override could not be set to 1")
+    aperturef_override_set_to_0 = (
+        "ApertureF Override set to 0",
+        "ApertureF Override could not be set to 0")
+    focusdistance_override_set_to_0 = (
+        "FocusDistance Override set to 1",
+        "FocusDistance Override could not be set to 1")
+    focusdistance_override_set_to_1 = (
+        "FocusDistance Override set to 0",
+        "FocusDistance Override could not be set to 0")
+    enableautofocus_override_on = (
+        "EnableAutoFocus Override turned on",
+        "EnableAutoFocus Override could not be turned on")
+    enableautofocus_override_off = (
+        "EnableAutoFocus Override turned off",
+        "EnableAutoFocus Override could not be turned off")
```

Comment:
these can be combined into one tuple "EnableAutoFocus property toggled", "EnableAutoFocus property failed to toggle"